[2024-05-27T07:48:28.651+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-27T07:48:28.761+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_parse_store_report_weather_data.fetch_and_save_weather_data scheduled__2024-05-10T22:00:00+00:00 [queued]>
[2024-05-27T07:48:28.785+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_parse_store_report_weather_data.fetch_and_save_weather_data scheduled__2024-05-10T22:00:00+00:00 [queued]>
[2024-05-27T07:48:28.786+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-27T07:48:28.853+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_and_save_weather_data> on 2024-05-10 22:00:00+00:00
[2024-05-27T07:48:28.886+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2191) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-27T07:48:28.889+0000] {standard_task_runner.py:63} INFO - Started process 2337 to run task
[2024-05-27T07:48:28.893+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_parse_store_report_weather_data', 'fetch_and_save_weather_data', 'scheduled__2024-05-10T22:00:00+00:00', '--job-id', '10983', '--raw', '--subdir', 'DAGS_FOLDER/cris.py', '--cfg-path', '/tmp/tmp68u8op8h']
[2024-05-27T07:48:28.903+0000] {standard_task_runner.py:91} INFO - Job 10983: Subtask fetch_and_save_weather_data
[2024-05-27T07:48:29.039+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_parse_store_report_weather_data.fetch_and_save_weather_data scheduled__2024-05-10T22:00:00+00:00 [running]> on host b151419bfa2a
[2024-05-27T07:48:29.283+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_parse_store_report_weather_data' AIRFLOW_CTX_TASK_ID='fetch_and_save_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-10T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-10T22:00:00+00:00'
[2024-05-27T07:48:29.285+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-27T07:49:02.687+0000] {logging_mixin.py:188} INFO - Data has been successfully saved to the JSON file
[2024-05-27T07:49:02.689+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-27T07:49:02.690+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-27T07:49:02.895+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=fetch_parse_store_report_weather_data, task_id=fetch_and_save_weather_data, execution_date=20240510T220000, start_date=20240527T074828, end_date=20240527T074902
[2024-05-27T07:49:02.968+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-27T07:49:03.019+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-27T07:49:03.026+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
